layers/+lang/lua/funcs.el

```diff
@@ -33,13 +39,19 @@
 
 
 ;; LSP Lua
-(defun spacemacs//lua-setup-lsp-emmy ()
-  "Setup LSP Lua."
-  (when lua-lsp-emmy-java-path
-    (setq lsp-clients-emmy-lua-java-path lua-lsp-emmy-java-path))
-  (when lua-lsp-emmy-jar-path
-    (setq lsp-clients-emmy-lua-jar-path (expand-file-name lua-lsp-emmy-jar-path)))
-  (setq lsp-enable-file-watchers lua-lsp-emmy-enable-file-watchers)
+(defun spacemacs//lua-setup-lsp-emmy-lua ()
```

Comment:
These three functions can be combined.
For setting `lsp-disable-clients`, these actually are not needed.

See comments above.